source/documentation/test-scenarios.html.md.erb

```diff
@@ -76,3 +76,77 @@ A 503 Service Unavailable error response will take the form of
   "emcsCorrelationId": "3e8dae97-b586-4cef-8511-68ac12da9028"
 }
 ```
+
+## Endpoint: Submit a message for an excise movement
+
+You can test [this endpoint](/api-documentation/docs/api/service/excise-movement-control-system-api/1.0/oas/page#tag/Movements/operation/postMessage) in the External Test environment with several different messages.
+
+### IE818 - Report of receipt
+
+You will need to provide a request with an XML body that conforms to the [IE818 XML Schema](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie818-v16-0.xsd).
+
+Within the IE815 XML body you can send the following values as Complementary Information to simulate responses:
+
+
+| Complementary Information    | 	Response                 |
+|:-----------------------------|:--------------------------|
+| SendA400                     | 400 Bad Request           |
+| SendA404                     | 404 Not Found             |
+| SendA500                     | 500 Internal Server Error |
+| SendA503                     | 503 Service Unavailable   |
+| Any other valid value        | 200 Success               |
```

Comment:
Thanks, this might need updating in the [readme](https://github.com/hmrc/emcs-api-eis-stub/blob/main/README.md)? It says IE815 and IE818 will return a 200.